docs/references/concepts.md

```diff
@@ -4,16 +4,35 @@ description: Fundamental user-facing concepts
 
 # Key Concepts
 
+## Instance
+
+An "instance" of Opstrace runs inside your cloud account.
+It is installed with the CLI `opstrace create` command.
+This name must be _globally_ unique and limited to 23 characters (`[a-z0-9-_]`) because it used to provide a conveient DNS name:  `<instance_name>.opstrace.io`.
```

Comment:
> conveient

Reply:
Personally, I think we should say a few things about that an Opstrace instance is comprised of multiple machines/nodes,   a Kubernetes cluster, and a bunch of cloud resources. Just to architecturally make it _clear_ that the instances entails _everything_ (or -- does it? important architectural question to be answered in the mid term)

Reply:
Let's defer the architecture bits for now.  I think it's a good idea, but I think that is a larger conversation than can be had here at the moment.